.github/workflows/mql-mimic-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: MQL Mimic Tests
+
+on:
+  push:
+    branches: [ "**" ]
+
+concurrency:
+  # For pull_request_target workflows we want to use head_ref -- the branch triggering the workflow. Otherwise,
+  # use ref, which is the branch for a push event.
```

Comment:
this comment doesn't seem to reflect the code?

Reply:
👍 copy/paste situation, will cleanup.